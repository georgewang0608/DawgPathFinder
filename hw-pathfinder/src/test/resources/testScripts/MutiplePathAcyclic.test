# Find a path with Dijkstra's algorithm when
# there exist multiple paths between the nodes in an acyclic graph

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D
AddNode simpleGraph E

AddEdge simpleGraph A B 2
AddEdge simpleGraph A C 2
AddEdge simpleGraph A D 1
AddEdge simpleGraph B E 1
AddEdge simpleGraph C E 1
AddEdge simpleGraph D E 1

FindPath simpleGraph A E