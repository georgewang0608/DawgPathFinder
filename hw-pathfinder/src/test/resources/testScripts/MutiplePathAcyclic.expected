# Find a path with Dijkstra's algorithm when
# there exist multiple paths between the nodes in an acyclic graph

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph
added node E to simpleGraph

added edge 2.000 from A to B in simpleGraph
added edge 2.000 from A to C in simpleGraph
added edge 1.000 from A to D in simpleGraph
added edge 1.000 from B to E in simpleGraph
added edge 1.000 from C to E in simpleGraph
added edge 1.000 from D to E in simpleGraph

path from A to E:
A to D with weight 1.000
D to E with weight 1.000
total cost: 2.000